src/pages/home/report/EmojiSkinToneList.js

```diff
@@ -0,0 +1,108 @@
+import React, {useState} from 'react';
+import {View, Pressable} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../../styles/styles';
+import compose from '../../../libs/compose';
+import {skinTones} from '../../../../assets/emojis';
+import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import Text from '../../../components/Text';
+import getSkinToneEmojiFromIndex from './EmojiPickerMenu/getSkinToneEmojiFromIndex';
+import EmojiPickerMenuItem from './EmojiPickerMenuItem';
+
+const propTypes = {
+
+    /** Visibility flag for skin tone picker */
+    isSkinToneListVisible: PropTypes.bool,
```

Comment:
Why is this controlled by props instead of just being controlled with state locally in this component?

Reply:
Because the trigger will always be from the main picker component. 

Still, want to maintain it in the state?

Reply:
Why is the trigger in the main picker component though? Can't we just put all of that logic in here?

Reply:
Don't remember why I did it that way. Moved to state.